README.md

```diff
@@ -43,28 +43,14 @@ Prerequisites:
 - [Virtual Box](https://www.virtualbox.org/)
 
 
-Then install the Ansible libraries for Postgresql and NodeJS
-
-`ansible-galaxy install geerlingguy.nodejs`
-
-`ansible-galaxy install ANXS.postgresql`
-
-Now you need to setup your virtual machine. 
+Now you need to setup your virtual machine and run the application
```

Comment:
Maybe the Vagrant VM use them for the installation, but it's not required for your machine